### ISSUE TITLE
Latent Consistency Models Support

### ISSUE THREAD

**Issue by julien-blanchon** [huggingface/candle#1331]
## Brief introduction to Latent Consistency Models

**Latent Consistency Models** (LCMs) are a novel approach in image synthesis that **enhance pre-trained Latent Diffusion Models**, such as Stable Diffusion 1.5 / SDX, by enabling high-resolution image generation with significantly fewer diffusion steps. 
They achieve this by predicting the solution of the underlying ODE instead of predicting the score function and then propagate it.

Here is the main page of the paper: https://latent-consistency-models.github.io/
The paper repo is: https://github.com/luosiallen/latent-consistency-model

## Current state

The original version of LCMs could be used with two method:
- By **applying a LoRA** to a pretrain Latent Diffusion Model (LDM), available here: [Latent Consistency Models LoRAs](https://huggingface.co/collections/latent-consistency/latent-consistency-models-loras-654cdd24e111e16f0865fba6)
- By directly using the **Standalone LCMs** version, available here: [Latent Consistency Models Weights](https://huggingface.co/collections/latent-consistency/latent-consistency-models-weights-654ce61a95edd6dffccef6a8)

Currently Standalone LCMs are already available in 🤗 Diffusers under [LCM Pipeline](https://github.com/huggingface/diffusers/tree/main/src/diffusers/pipelines/latent_consistency_models) and [LCM Scheduler](https://github.com/huggingface/diffusers/blob/main/src/diffusers/schedulers/scheduling_lcm.py). 

A pretrain version of LCM on SSD1B is available and achieve very good result, I would like to implement it here in Candle because this might enable a fast image generation directly in browser with WASM.

Are you guys interested in implementing LCM here ? The most direct way would be to implement Stable Diffusion LoRA, so it will also enable the usage of any custom LoRA
